migrator.go

```diff
@@ -281,7 +280,22 @@ func (m Migrator) AlterColumn(value interface{}, field string) error {
 			}
 
 			fileType := clause.Expr{SQL: m.DataTypeOf(field)}
+			// check for typeName and SQL name
+			isSameType := true
 			if fieldColumnType.DatabaseTypeName() != fileType.SQL {
+				isSameType = false
+				// if different, also check for aliases
+				aliases := m.GetTypeAliases(fieldColumnType.DatabaseTypeName())
```

Comment:
This fix is to consider the "field type aliases" when checking if the column type has changed.